modules/virt-hot-unplugging-a-virtual-disk-using-the-web-console.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
```

Comment:
assembly path should include the virt folder. 

_virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc_

Same comment applies to the other module